src/dataflow/utilities/node.ts

```diff
@@ -230,3 +233,14 @@ export interface NodeChannelInfo {
 export const roundNodeValue = (n: number) => {
   return Math.round(n * 1000) / 1000;
 };
+
+export const NodePlotColors = ["#0592AF", "#FFB399", "#FF33FF", "#FFFF99", "#00B3E6",
+                               "#E6B333", "#3366E6", "#999966", "#99FF99", "#B34D4D",
+                               "#80B300", "#809900", "#E6B3B3", "#6680B3", "#66991A",
+                               "#FF99E6", "#CCFF1A", "#FF1A66", "#E6331A", "#33FFCC",
+                               "#66994D", "#B366CC", "#4D8000", "#B33300", "#CC80CC",
+                               "#66664D", "#991AFF", "#E666FF", "#4DB3FF", "#1AB399",
+                               "#E666B3", "#33991A", "#CC9999", "#B3B31A", "#00E680",
+                               "#4D8066", "#809980", "#E6FF80", "#1AFF33", "#999933",
+                               "#FF3380", "#CCCC00", "#66E64D", "#4D80CC", "#9900B3",
+                               "#E64D66", "#4DB380", "#FF4D4D", "#99E6E6", "#6666FF"];
```

Comment:
This color array is used to provide different colors for the graph traces in the data storage node.  The first color in this array is the graph line color used in DF 2.  Other colors are from some random color list found online.  Not sure how many of these we need or what the colors should be (this list is probably longer than needed, but we haven't discussed capping the number of inputs).

Reply:
Adding your comment to the code with a URL would be helpful If you have it or can find it.

Reply:
I decided to swap these out and use a list of colors that Michael created for Cbio.

Reply:
👍